tasks/install.yml

```diff
@@ -11,25 +11,6 @@
     append: yes
     createhome: yes
 
-- name: Add Python PPA
-  apt_repository:
-    repo: "{{ django_apt_python_ppa }}"
-    state: present
-  become: true
-  become_user: root
-  when: django_apt_python_ppa is defined and django_apt_python_ppa != ""
-
-- name: Update apt cache
-  apt:
-    update_cache: "yes"
-
-- name: Install system-wide dependencies
-  apt:
-    name: "{{ django_system_wide_dependencies }}"
-    state: latest
-    update_cache: yes
-    cache_valid_time: 600
-
 - name: Delete virtualenv
```

Comment:
Has this been moved or it is no longer necessary?

Reply:
There might be Django dependencies that may not be applicable to python in general, for example installing zlib1g-dev which may be a dependency on your Django application for the requirements to be successfully installed.

Reply:
This was moved. Mhh I see, that make sense didn't think about it that way. I'll revert this back.

Reply:
Reverted the `Install system-wide dependencies`. I've still left out the `Add Python PPA` tasks since I think it's better to have within the role that installs python.... thoughts ? @ukanga